netdisco/discoverables/apple_tv.py

```diff
@@ -13,5 +14,6 @@ def __init__(self, nd):
     def info_from_entry(self, entry):
         """Returns most important info from mDNS entries."""
         info = super().info_from_entry(entry)
-        info[ATTR_NAME] = info[ATTR_PROPERTIES]['name'].replace('\xa0', ' ')
+        info[ATTR_HOST] = str(ipaddress.ip_address(entry.address))
```

Comment:
Looks like your implementation returns the same info as the existing implementation on my machine but yours looks cleaner, can you add it to the MDNS base class?